dojo/tools/yarn_audit/parser.py

```diff
@@ -44,6 +48,38 @@ def get_items_yarn(self, tree, test):
                 )
         return list(items.values())
 
+    def get_items_yarn2(self, tree, test):
+        items = []
+        for element in tree:
+            value = "**value:** " + str(element.get("value"))
```

Comment:
I don't think we should include Markdown formatting in `component_name` like this. Additionally, this will set the component name to "None" instead of leaving it empty if there's no `value`. I'd check if `value` is non-empty, and if so, include the `str()` value as-is.